README.md

```diff
@@ -47,9 +47,11 @@ First, you need to prepare your configuration file for the guest VM (several exa
 ```console
 $ cd arceos-vmm
 # x86_64
-$ make A=$(pwd) ACCEL=y BLK=y ARCH=x86_64 [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
-# aarch64 & riscv64
-$ make A=$(pwd) ACCEL=n BLK=y ARCH=[aarch64|riscv64] [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
+$ make ACCEL=y ARCH=x86_64 [LOG=warn|info|debug|trace] VM_CONFIGS=/PATH/TO/CONFIG/FILE run
+# aarch64
+$ make ACCEL=n ARCH=aarch64 [LOG=warn|info|debug|trace] PLATFORM=aarch64-qemu-virt-hv VM_CONFIGS=/PATH/TO/CONFIG/FILE run
```

Comment:
Can we add a brief comment here to explain why `PLATFORM` is necessary here for aarch64?

Reply:
As a matter of fact, I specify it in Makefile on my next PR.